scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -45,15 +49,63 @@ public static BuildTargetResult InitPublish(BuildTargetContext c)
         [Environment("PUBLISH_TO_AZURE_BLOB", "1", "true")] // This is set by CI systems
         public static BuildTargetResult Publish(BuildTargetContext c)
         {
+            if (ShouldPublishToLatest())
+            {
+                string targetContainer = $"{Channel}/Binaries/Latest/";
+
+                foreach (string blob in _latestBlobs)
+                    AzurePublisherTool.CopyBlob(blob.Replace("/dotnet/", ""), (targetContainer + Path.GetFileName(blob)).Replace(CliNuGetVersion, "latest").Replace(SharedFrameworkNugetVersion, "latest"));
+
+                PublishLatestCliVersionTextFile(c);
+                PublishLatestSharedFrameworkVersionTextFile(c);
+            }
+
             return c.Success();
         }
 
+        private static bool ShouldPublishToLatest()
+        {
+             Dictionary<string, bool> runtimes = new Dictionary<string, bool>()
```

Comment:
It's not too clear that this code is checking that we have runtime packages for each of these rids, maybe make the function name more descriptive?